### DISCUSSION TITLE
GC crash in WKS::gc_heap::background_mark_simple (.Net Framework 4.8)

### DISCUSSION THREAD

**Discussion by levicki** [dotnet/runtime#105750] [Q&A] [answered]
@mangod9 Can you please let me know whether the bugfix for [this issue](https://github.com/dotnet/runtime/issues/81154) has been backported to .Net Framweork 4.8 / 4.8.1? If yes, starting from which version / build number? If no, any chance of reconsidering fixing it?

**Comment by kazepis** [comment]
I would like to know this as well. Thank you in advance!

**Comment by mangod9** [comment, marked as answer]
Hey @levicki , the specific issue you refer to shouldnt occur in Framework since that doesnt have the mark prefetch functionality. It could be some other heap corruption. For framework issues I would suggest creating an issue via https://developercommunity.visualstudio.com/dotnet

**Reply by levicki** [reply]
Thanks!
